app/src/main/java/net/squanchy/schedule/ScheduleService.kt

```diff
@@ -1,88 +1,100 @@
 package net.squanchy.schedule
 
 import io.reactivex.Observable
-import io.reactivex.functions.Function3
-import net.squanchy.schedule.domain.view.Event
+import io.reactivex.functions.BiFunction
+import io.reactivex.schedulers.Schedulers
 import net.squanchy.schedule.domain.view.Schedule
 import net.squanchy.schedule.domain.view.SchedulePage
 import net.squanchy.schedule.domain.view.Track
+import net.squanchy.schedule.filterschedule.TracksFilter
 import net.squanchy.service.firebase.FirebaseAuthService
 import net.squanchy.service.firestore.FirestoreDbService
+import net.squanchy.service.firestore.model.schedule.FirestoreEvent
 import net.squanchy.service.firestore.model.schedule.FirestoreSchedulePage
 import net.squanchy.service.firestore.toEvent
-import net.squanchy.service.firestore.toTrack
-import net.squanchy.service.repository.FilterScheduleRepository
 import net.squanchy.support.lang.Checksum
-import net.squanchy.support.lang.Optional
 import org.joda.time.DateTimeZone
 import org.joda.time.LocalDate
 
 interface ScheduleService {
-    fun schedule(onlyFavorites: Boolean): Observable<Schedule>
+    fun schedule(onlyFavorites: Boolean = false): Observable<Schedule>
```

Comment:
Added default value so we can call `schedule()` when we don't need to filter by favourites